app/controllers/admin/employees_controller.rb

```diff
@@ -84,10 +86,6 @@ def error_update_employee
       redirect_to edit_admin_employee_path
     end
 
-    def adm?
```

Comment:
I suggest you keep this method (and change method name to `is_admin?` to follow convention)

you can access `@employee` from this method and return true depending on its role.